livekit-api/livekit/api/_service.py

```diff
@@ -15,8 +15,14 @@ def __init__(
         self.api_key = api_key
         self.api_secret = api_secret
 
-    def _auth_header(self, grants: VideoGrants) -> Dict[str, str]:
-        token = AccessToken(self.api_key, self.api_secret).with_grants(grants).to_jwt()
+    def _auth_header(
+        self, grants: VideoGrants, sip: SIPGrants = None
```

Comment:
```suggestion
        self, grants: VideoGrants, sip: SIPGrants | None = None
```

Reply:
After adding it, the type checker complains about requiring Python 3.10. So it's probably worth doing this update separately.

Reply:
Uh, no, type check doesn't like it this way either... Let's see how to bump to Python 3.10.